molecule/default/converge.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+
+  vars:
```

Comment:
This role will need all the variables documented in the [example](https://github.com/AcroMedia/ansible-role-virtual-host#example-playbook), as well as the other [requierements](https://github.com/AcroMedia/ansible-role-virtual-host#requirements) before it will pass tests.